spec/spec_helper_local.rb

```diff
@@ -28,15 +30,24 @@ def with_openbsd_facts
   # operatingsystemrelease may contain X.X-current
   # or other prefixes
   let :facts do
-    super().merge(kernelversion: '5.9',
-                  osfamily: 'OpenBSD')
+    super().merge(
+      kernelversion: '5.9',
+      osfamily: 'OpenBSD'
+    )
   end
 end
 
 def with_redhat_facts
   let :facts do
-    super().merge(operatingsystemmajrelease: '7',
-                  osfamily: 'Redhat')
+    super().merge(
```

Comment:
is this what you had in mind? I think the previous layout may have just been what `rubocop -a` did, or else copying the example of what was there before.

Reply:
Pushed one more update, let me know if it looks right. It didn't look right when I indented the curly brace after the newline.

Reply:
As you can probalby see, I also updated the Debian / OpenBSD facts.

But agree that we should rework this whole thing at some point.